cspell.config.yaml

```diff
@@ -7,18 +7,20 @@ words:
   - databroker
   - deepcopy
   - envtest
+  - filemgr
   - mockgen
+  - oidc
   - pomerium
   - protobuf
-  - oidc
+  - protojson
   - readyz
   - sharedkey
   - sslcert
   - sslkey
   - sslrootcert
-  - upsert
   - uifs
-  - filemgr
+  - unmarshalled
```

Comment:
spelling nit: the [documentation](https://pkg.go.dev/encoding/json) for `encoding/json` uses "marshaled" with one l

(totally optional, I don't much care either way)